messagerie/patterns/404.php

```diff
@@ -48,4 +48,4 @@
 <!-- /wp:group -->
 
 <!-- wp:template-part {"slug":"footer","area":"footer"} /--></div>
-<!-- /wp:group -->
\ No newline at end of file
+<!-- /wp:group -->
```

Comment:
There are a couple of changes like this to Messagerie that I can't seem to revert. I think my linter is fixing some spacing issues on a pre-commit hook, it seems to run even if I skip the hook. They're only spacing changes so I think they're OK to keep.